index.html

```diff
@@ -0,0 +1,269 @@
+      <article class="feature__item">
+        <img class="feature__image" src="img/screen-3.png" alt="feature-image">
+        <h3 class="feature__heading">
+          FULLY LAYERED PSD
+        </h3>
+        <p class="feature__text text">
+          Lorem ipsum dolor sit amet consectetur adipiscing elit.
+        </p>
+        <a class="feature__link" href="#">
+          Check this out >
+        </a>
+      </article>
+      <article class="feature__item">
+        <img class="feature__image" src="img/screen-4.png" alt="feature-image">
+        <h3 class="feature__heading">
+          FULL COMPATIBILITY
+        </h3>
+        <p class="feature__text text">
+          Lorem ipsum dolor sit amet consectetur adipiscing elit.
+        </p>
+        <a class="feature__link" href="#">
+          Check this out >
+        </a>
+      </article>
+      <article class="feature__item">
+        <img class="feature__image" src="img/screen-5.png" alt="feature-image">
+        <h3 class="feature__heading">
+          VERY GREAT MODULES
+        </h3>
+        <p class="feature__text text">
+          Lorem ipsum dolor sit amet consectetur adipiscing elit.
+        </p>
+        <a class="feature__link" href="#">
+          Check this out >
+        </a>
+      </article>
+    </div>
+  </section>
+
+  <section class="actions">
+    <div class="actions__content container">
+      <article class="actions__item">
+        <h3 class="actions__heading">
+          <i class="fa fa-paper-plane actions__icon" aria-hidden="true"></i>SEND IT NOWcs
+        </h3>
+        <p class="actions__text text">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse
+          fringilla fringilla nisl congue congue.
+        </p>
+        <button class="actions__button button actions__button-text">
+          Read more
+        </button>
+      </article>
+      <article class="actions__item">
+        <h3 class="actions__heading">
+          <i class="fa fa-pencil actions__icon" aria-hidden="true"></i>VERY NICE TO SEE
+        </h3>
+        <p class="actions__text text">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse
+          fringilla fringilla nisl congue congue.
+        </p>
+        <button class="actions__button button">
+          Read more
+        </button>
+      </article>
+    </div>
+  </section>
+
+  <section class="themeforest">
+    <div class="container themeforest__content">
+      <section class="themeforest__item">
+        <div class="image-container">
+          <img class="themeforest__image" src="img/screen-6.png" alt="actions-image" />
+        </div>
+      </section>
+      <section class="themeforest__item" >
+        <h3 class="themeforest__heading">
+          JUST FOR YOU ON THEMEFOREST
+        </h3>
+        <p class="themeforest__text text">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec interdum mollis risus, in mattis erat convallis egestas.
```

Comment:
Max line length should be 80 characters.
Please check your index.html file and correct it manually or use js-beautify.

Reply:
Done, Please check.

Reply:
You didn't change max line length to 80 characters. Please check index.html file again.

Reply:
I did, but js-beautify changed it to the previous variant... I updated js-beautify settings. Please, check again.